styles/globals.css

```diff
@@ -67,3 +67,9 @@ footer a {
   min-height: 92vh;
   padding-bottom: 50px;
 }
+.d-none {
+  display: none !important;
+}
+.d-block {
+  display: block !important;
```

Comment:
Can we please remove `!important` here as well ??

Reply:
Removed `d-block` class as it was not used anywhere. Regarding `!important` property in `d-none` class, I could not find a better way around it. So, I think we can keep it